core-processor/src/ext.rs

```diff
@@ -458,7 +465,7 @@ impl EnvExt for Ext {
         self.charge_gas_runtime(RuntimeCosts::Debug)?;
 
         if data.starts_with("panic occurred") {
```

Comment:
Could we strip our custom "panic occurred"?

Reply:
We can but should we? This will be printed as the whole message in logs

Reply:
I mean we use "panic occurred" insertion for detecting that there is the Rust panic, not only wasm unreachable statement, and after this we wrap it into `PanicOccured` and I would like to set display there:

`core-processor/src/ext.rs`
```
/// User's code panicked
#[display(fmt = "Panic occurred: {}", _0)]
Panic(String),
```

Reply:
I checked possible messages starting with `panic occurred` and what I found:

- `panic occurred: '{:?}', {}:{}:{}`
- `panic occurred, {}:{}:{}`
- `panic occurred`

It's quite annoying and unrobust to strip punctuation and whitspace chars so I'd leave it as is

Reply:
Maybe it will be better to introduce new syscall for panics in the future?

Reply:
I would wrap it like `panic occurred: "EXPLANATION"`, where explanation may be unknown, may contain or not file location etc and strip "panic occurred: "